packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -330,6 +336,19 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
   }
   private _disableOptimizedSrcset = false;
 
+  /**
+   * Sets the image to "fill mode," which eliminates the height/width requirement and adds
```

Comment:
nit: typo, let's fix it in a followup PR.
```suggestion
   * Sets the image to "fill mode", which eliminates the height/width requirement and adds
```